### DISCUSSION TITLE
Load from Json Different Screen Sizes

### DISCUSSION THREAD

**Discussion by juzimmer** [fabricjs/fabric.js#7320] [General]
Hello, I am trying to find a way to scale all objects and background image up or down dependent on the screen size currently in use. Right now I'm storing the screen width and height upon creation but I'm not completely sure how to go about doing this. The user can't do anything with zoom. Any ideas ?

**Comment by svoboda200786** [comment]
Hard to understand - put some code here. Are you storing the width and height you have at the begining? And then you compare current values with stored values? And it's not good?
